src/shitalSDec19/assignment3/Solution13.java

```diff
@@ -0,0 +1,30 @@
+/* WAP to find number is Armstrong or not. 
+ * a. Int number = 153 i. Number is Armstrong 
+ * b. Int number = 234 i. Number is not Armstrong  
+ * Note :- Armstrong number means sum of each number cubes to number it self. 
+  */
+
+package shitalSDec19.assignment3;
+import java.util.Scanner;
+public class Solution13 {
+	
+	static int sum = 0;
+	
+	
+	public static void main(String[] args) {
+		System.out.println("please provide no of iterations you want");
+		 Scanner sc = new Scanner (System.in);
+		int size = sc.nextInt();
+		for (int i=0; i<size; i++)
+			{
+				System.out.println("Enter number");
+				Scanner scan = new Scanner (System.in);
```

Comment:
why we use two time scanner class in same method.
please give explanation. @ShitalSDec19

Reply:
i have updated my program, please review.